pilot/pkg/bootstrap/monitoring.go

```diff
@@ -64,6 +65,8 @@ func addMonitor(mux *http.ServeMux) error {
 		}
 	})
 
+	zpages.Handle(mux, "/debug")
```

Comment:
should this be wrapped with an `if features.GRPCDebug`?

Reply:
What can zpages show?